backend/src/main.ts

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Entry point of the application.
+ * Configures and starts the server.
+ * @packageDocumentation
+ */
+import * as dotenv from "dotenv";
+import 'reflect-metadata';
+dotenv.config({ path: 'secrets.env' });
+dotenv.config();
+
+import { InversifyExpressServer } from 'inversify-express-utils';
+
+import container from "./backendmodule";
+import { Logger, PORT, serverConfig, serverErrorConfig } from "./config";
+import './controllers/AIAssistant.Controller';
+
+
+export async function Bootstrap() {
+  const server = new InversifyExpressServer(container);
+  server.setConfig(serverConfig);
+  process.env.NODE_ENV === "production" ? server.setErrorConfig(serverErrorConfig): null;
+
+  const app = server.build();
+  app.listen(PORT, () =>
+    new Logger().info(`Server up on http://127.0.0.1:${PORT}/`)
```

Comment:
I think server address should be also part of a config or env file and not hard written

Reply:
I am not entirely sure what you mean. The port is imported from the config file and reads `process.env` and defaults to port 3001 if nothing is in the environment. If you mean the ip address, then there is no host specified in the call, so it listens on all available interfaces. I see now that the logger explicitly only says that it is listening on localhost. If we really want to, we could change that log message to something like `Server up on port: ${PORT}`. However, I don't see much difference here, since it is just the log message